docs/support-bundle-guide.md

```diff
@@ -0,0 +1,253 @@
+# Support Bundle User Guide
+
+## What is Support Bundle
+
+Antrea supports collecting support bundle tarballs, which include the information
+from Antrea Controller and Antrea Agent. The collected information can help
+debugging issues in the Kubernetes cluster.
+
+**Be aware that the generated support bundle includes a lot of information,
+including logs, so please review the contents before sharing it on Github
+and ensure that you do not share any sensitive information.**
+
+There are two ways of generating support bundles. Firstly, you can run `antctl supportbundle`
+directly in the Antrea Agent Pod, Antrea Controller Pod, or on a host with a
+`kubeconfig` file for the target the cluster. Secondly, you can also apply
+`SupportBundleCollection` CRs to create support bundles for Kubernetes Nodes
```

Comment:
I saw you use K8s in other places. Then probably change this one too

Reply:
updated.